pkg/harvester/config/feature-flags.js

```diff
@@ -0,0 +1,54 @@
+
+// https://github.com/harvester/harvester/wiki/Roadmap
+const FEATURES = {
+  cpuPinning:                                     false,
+  usbPassthrough:                                 false,
+  volumeEncryption:                               false,
+  schedulingVMBackup:                             false,
+  vmSnapshotQuota:                                false,
+  longhornV2LVMSupport:                           false,
+  improveMaintainMode:                            false,
+  autoRotateRke2CertsSetting:                     false,
+  kubeconfigDefaultTokenTTLMinutesSetting:        false,
+  supportBundleNodeCollectionTimeoutSetting:      false
+};
+
+// https://github.com/harvester/dashboard/releases/tag/v1.3.0
+const releaseV130 = { ...FEATURES };
```

Comment:
Ideally, we should remove it to prevent the file from growing larger?
we can use general naming to prevent this.
```js
const features = { ...FEATURES };
const releaseV131 = {
  ...features,
  autoRotateRke2CertsSetting: true,
  supportBundleNodeCollectionTimeoutSetting: true
};
const releaseV132 = {
  ...features,
  autoRotateRke2CertsSetting: true,
  supportBundleNodeCollectionTimeoutSetting: true,
  kubeconfigDefaultTokenTTLMinutesSetting: true,
};
```

here is another idea, the result is the same
```js
export const RELEASE_FEATURES = {
  "v1.3.1": ["autoRotateRke2CertsSetting", "supportBundleNodeCollectionTimeoutSetting"],
  "v1.3.2": ["kubeconfigDefaultTokenTTLMinutesSetting"],
  "v1.4.0": ["cpuPinning", ...]
  },
};

const getFeatures = (version) => {
  const releasedFeatures = RELEASE_FEATURES[version] || [];
  const features = { ...FEATURES };

  releasedFeatures.forEach((feature) => features[feature] = true;
  return features;
};
```

Reply:
The first one is clear at a glance. Will change to that.